apps/web/pages/v2/apps/index.tsx

```diff
@@ -32,11 +32,9 @@ export const getServerSideProps = async (context: NextPageContext) => {
     appStore = await getAppRegistry();
   }
 
-  const categoryQuery = await prisma.app.findMany({
-    select: {
-      categories: true,
-    },
-  });
```

Comment:
Removed getting all categories from the database in favor of filtering down the categories existent in the available apps returned by the App Registry.